### PR TITLE
Vpnaas: IKE Policy resource

### PR DESCRIPTION
**Pull request by simonre**
Part 3 out of 5 of a series that addresses #2

### CODE REVIEW

**Comment by simonre** [comment]
@jtopjian This is ready for review

**Comment by jtopjian** [comment]
@simonre Looks good!

Just want to confirm: the IPSec resource has `WaitForState`'s for Create and Update, but this resource doesn't. Is that intentional?

**Comment by simonre** [comment]
@jtopjian No it's not, I'll add it in.

**Comment by simonre** [comment]
@jtopjian I added the waitforstates

**Comment by jtopjian** [comment]
recheck

File: website/docs/r/vpnaas_ike_policy_v2.html.markdown [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+layout: "openstack"
+page_title: "OpenStack: openstack_vpnaas_ike_policy_v2"
+sidebar_current: "docs-openstack-resource-vpnaas-ike-policy-v2"
+description: |-
+  Manages a V2 Neutron IKE policy resource within OpenStack.
+---
+
+# openstack\_vpnaas\_ike_policy_v2
```
Comment (jtopjian) [inline website/docs/r/vpnaas_ike_policy_v2.html.markdown]: `openstack\_vpnaas\_ike\_policy_v2`

**Pull request merged**